w3-replication.md

```diff
@@ -0,0 +1,397 @@
+# Replication Protocol
```

Comment:
I propose we settle on names to differentiate the "node with the data" and the "node replicating the data". Might be worth explicitly defining this in the introduction, or language section?

Primary Node - The node that originally received and stored the initial blob upload
Replica Node - Any node that receives a copy of the blob during the replication process